pkg/diffs/diffs.go

```diff
@@ -0,0 +1,68 @@
+package diffs
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/equality"
+	prowconfig "k8s.io/test-infra/prow/config"
+)
+
+// Diffs holds the information where the differences can be found.
+type Diffs struct {
+	ConfigPath    string
+	JobConfigPath string
+}
+
+// NewDiffs creates a new Diffs{}
+func NewDiffs(configPath, jobConfigPath string) *Diffs {
+	return &Diffs{
+		ConfigPath:    configPath,
+		JobConfigPath: jobConfigPath,
+	}
+}
+
+// GetChangedPresubmits returns a mapping of repo to presubmits to execute.
+func (d *Diffs) GetChangedPresubmits() (map[string][]prowconfig.Presubmit, error) {
+	ret := make(map[string][]prowconfig.Presubmit)
+
+	prowMasterConfig, err := prowconfig.Load(d.ConfigPath, d.JobConfigPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to load master's Prow config: %v", err)
+	}
+
+	prowPRConfig, err := prowconfig.Load("./cluster/ci/config/prow/config.yaml", "./ci-operator/jobs/")
```

Comment:
Err - why are these hard-coded?

Reply:
shall we also include the `decorationPath` as before?

Reply:
Let's not do this.